docs/content/config/debugging.md

```diff
@@ -47,8 +47,7 @@ To get a shell inside the container run: `docker exec -it <CONTAINER NAME> bash`
 
 If you need more flexibility than what the `docker logs` command offers, then the most useful locations to get relevant DMS logs within the container are:
 
-- `/var/log/mail/mail.log`
```

Comment:
This one still exists and doesn't represent a single service AFAIK? Is it worth mentioning?

https://github.com/docker-mailserver/docker-mailserver/blob/218b56b123b6b74d6ac604db37ba21ae00090f2b/Dockerfile#L180

https://github.com/docker-mailserver/docker-mailserver/blob/218b56b123b6b74d6ac604db37ba21ae00090f2b/Dockerfile#L245

Reply:
Can probably be removed. I will provide a PR.